src/components/application_manager/test/message_helper/message_helper_test.cc

```diff
@@ -956,6 +957,115 @@ TEST_F(MessageHelperTest, SubscribeApplicationToSoftButton_CallFromApp) {
       message_params, appSharedPtr, function_id);
 }
 
+TEST(MessageHelperTestCreate,
+   CreateHashUpdateNotification_FunctionId_Equal) {
+  uint32_t app_id = 0;
+  smart_objects::SmartObjectSPtr ptr =
+      MessageHelper::CreateHashUpdateNotification(app_id);
+
+  EXPECT_TRUE(ptr);
+
+  smart_objects::SmartObject& obj = *ptr;
+
+  int function_id =
+      static_cast<int>(mobile_apis::FunctionID::OnHashChangeID);
+  int notification = static_cast<int>(::application_manager::kNotification);
+
+  EXPECT_EQ(function_id, obj[strings::params][strings::function_id].asInt());
+  EXPECT_EQ(app_id, obj[strings::params][strings::connection_key].asUInt());
+  EXPECT_EQ(notification,
+      obj[strings::params][strings::message_type].asInt());
+}
+
+TEST(MessageHelperTestCreate, CreateAppVrHelp_AppName_Equal) {
+  MockApplicationSharedPtr appSharedMock = utils::MakeShared<MockApplication>();
+
+  application_manager::CommandsMap vis;
+  DataAccessor< ::application_manager::CommandsMap>
+      data_accessor(vis, true);
+
+  const smart_objects::SmartObject* objPtr = NULL;
```

Comment:
@Kozoriz 
Also I would prefer to have name like vr_synonyms_ptr instead of objPtr